Sources/App/Icon/IconVariable.swift

```diff
@@ -220,6 +222,8 @@ enum IconSurfaceVariable: String, CaseIterable, GenericVariableMixable {
         case .cloud_cover_low: return 1
         case .cloud_cover_mid: return 1
         case .cloud_cover_high: return 1
+        case .convective_cloud_top: return 1
+        case .convective_cloud_base: return 1
```

Comment:
A scale factor of `1` might not be ideal. Underlaying compression support a value of up to 32767 and 32 km of should not be possible for cloud levels, however, a precision of 1 metre seems a bit high which leads to large database files. 

Can you set the scale factor to 0.1 which has a precision of 10 metres? For cloud base and top in the DMI model, I am using 0.05 (20 metre precision). Because convective clouds should be lower, 0.1 (10 metre) should be fine.

Reply:
Sure!